onnx/backend/test/case/node/ai_onnx_ml/binarizer.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+import onnx
+
+from ...base import Base
+from .. import expect
+
+
+def compute_binarizer(x, threshold=None):
+    y = x.copy()
+    cond = y > threshold
+    not_cond = np.logical_not(cond)
+    y[cond] = 1
+    y[not_cond] = 0
+    return (y,)
```

Comment:
Would `return np.where(x > threshold, 0, 1)` work? Or, `return (x <= threshold).astype(int)`, using whatever desired target type is (int32 or int64).

Reply:
Ok, I see the output type needs to be same as input type. So, perhaps `return (x < threshold).astype(x.dtype)` or something like that? I see this comes from the reference implementation, so anyway, this is a suggestion for that. As mentioned above, we can directly reference that from here.

Reply:
code updated according to @gramalingam 's recommendation